### DISCUSSION TITLE
Istio Ambient mode

### DISCUSSION THREAD

**Discussion by anuraj-narayanan** [istio/istio#52823] [Q&A] [answered]
How can I deploy the waypoint proxy using Helm? Didn't see any option with helm for the equivalent istioctl command 'istioctl x waypoint apply --enroll-namespace --wait '

**Comment by howardjohn** [comment, marked as answer]
`istioctl waypoint apply` is only a _very_ small helper. `istioctl manifest generate` can output the plain YAML you can apply with any tooling:

```
$ istioctl waypoint generate
apiVersion: gateway.networking.k8s.io/v1
kind: Gateway
metadata:
  name: waypoint
spec:
  gatewayClassName: istio-waypoint
  listeners:
  - name: mesh
    port: 15008
    protocol: HBONE
```

Note the --enroll-namespace part is equivilent to `kubectl label namespace <namespace> istio.io/use-waypoint=<gateway name>`

**Reply by anuraj-narayanan** [reply]
Thank you @howardjohn

**Comment by anuraj-narayanan** [comment]
@howardjohn 
Can you please advise why the waypoint deployment is not considering the image docker.io/istio/proxyv2:1.23.0 and taking the distroless image : docker.io/istio/proxyv2:1.23.0-distroless

**Reply by zirain** [reply]
for security reason, you should always use distroless image.

**Reply by howardjohn** [reply]
When you install ambient with `--set profile=ambient` it defaults to use disrtoless images. You can set `global.variant=""` to undo if you want.
